conf_files/pocs.yaml

```diff
@@ -52,4 +52,15 @@ messaging:
     # Must match ports in peas.yaml.
     cmd_port: 6500
     msg_port: 6510
+#Enable to output POCS messages to social accounts
+# social_accounts:
+#     twitter:
+#         consumer_key: [your_consumer_key]
+#         consumer_secret: [your_consumer_secret]
+#         access_token: [your_access_token]
+#         access_token_secret: [your_access_token_secret] 
+#     slack:
+#         webhook_url: [your_webhook_url]
+#         output_timestamp: False
```

Comment:
Should `output_timestamp` also appear in the twitter config?

Reply:
No that would not be allowed either. 
The rules are fuzzy but they are getting stricter. I'm not super familiar with Twitter but there is some info [here](https://twittercommunity.com/t/duplicate-tweets/13264) and [here](https://blog.smarterqueue.com/faq/march-2018-twitter-duplication-policy-is-changing/)

Reply:
Thanks for the clarification.